src/module-elasticsuite-core/Helper/IndexSettings.php

```diff
@@ -40,6 +41,28 @@ class IndexSettings extends AbstractConfiguration
      */
     const INDICES_SETTINGS_CONFIG_XML_PREFIX = 'indices_settings';
 
+    /**
+     * Location of Elasticsuite analysis settings configuration.
+     *
+     * @var string
+     */
+    const ANALYSIS_CONFIG_XML_PREFIX = 'smile_elasticsuite_core_analysis_settings';
+
+    /**
+     * @var string
+     */
+    const REFERENCE_ANALYZER_CONFIG_XML_PREFIX = 'reference_analyzer';
+
+    /**
+     * @var string
+     */
+    const OLD_DEFAULT_INDICES_PATTERN = '{{YYYYMMdd}}_{{HHmmss}}';
```

Comment:
do we need this in this commit ?

Reply:
Strange, this is something related to the cherry-pick from the based-on-2.11.x feature branch that I thought was removed.